ci/bamboo/build-manylinux-prototype.sh

```diff
@@ -1,101 +0,0 @@
-#!/bin/bash
```

Comment:
That seems reasonable. I think it is confusing that `nupic` uses `scripts/` for code-related things but we should probably move those things into the actual Python package and expose via setup.py `entry_points`.

In other words, that's fine but we may still want to have a script in `ci/bamboo/` that passes the Bamboo build number to the script for file outputs.